### ISSUE TITLE
Mojarra 4.0 WIP namespace issue with Hello World Example

### ISSUE THREAD

**Issue by raupachz** [eclipse-ee4j/mojarra#4903]
Hello there,

there is some minor issue with the Hello World Example application on the landing page. I deployed the example on Apache Tomcat 10.0.4. Unfortunately the outcome is not what I expected. 

Here is the `/hello.xhtml`:

```
<!DOCTYPE html>
<html lang="en"
    xmlns="http://www.w3.org/1999/xhtml"
    xmlns:f="jakarta.faces.core"
    xmlns:h="jakarta.faces.html">
    <h:head>
        <title>Hello, World!</title>
    </h:head>
    <h:body>
        <h:form>
            <h:outputLabel for="name" value="Enter your name" required="true" />
            <h:inputText id="name" value="#{hello.name}" />
            <h:message for="name" />
            <br />
            <h:commandButton value="Say hello" action="#{hello.createMessage}">
                <f:ajax execute="@form" render="@form" />
            </h:commandButton>
            <br />
            #{hello.message}
        </h:form>
    </h:body>
</html>
```

And here is the rendered response:

```
<!DOCTYPE html>
<html lang="en"
      xmlns="http://www.w3.org/1999/xhtml"
      xmlns:f="http://xmlns.jcp.org/jsf/core"
      xmlns:h="http://xmlns.jcp.org/jsf/html">
<h:head>
    <title>Hello, World!</title>
</h:head>
<h:body>
    <h:form>
        <h:outputLabel for="name" value="Enter your name" required="true" />
        <h:inputText id="name" value="#{hello.name}" />
        <h:message for="name" />
        <br />
        <h1>#{hello.message}</h1>
        <h:commandButton value="Say hello" action="#{hello.createMessage()}">
            <f:ajax execute="@form" render="@form" />
        </h:commandButton>
        <br />

    </h:form>
</h:body>
</html>
```

For some unknown reason the components are not rendered. The page itself shows up. I could verify in the debugger that the FacesServlet is executed. 

If one changes the XML namespaces back to Java EE 8 then everything works as expected and the response is just fine.

**tl;dr;**
```
xmlns:f="jakarta.faces.core"
xmlns:h="jakarta.faces.html"
```
does not work.
```
xmlns:f="http://xmlns.jcp.org/jsf/core"
xmlns:h="http://xmlns.jcp.org/jsf/html"
```
does work.

**Comment by christophs78** [comment]
Do you use Mojarra 3.0? (Not 4.0-SNAPSHOT)
Mojarra 3 still has the old Namespaces.
See https://github.com/eclipse-ee4j/mojarra/issues/4893 and https://github.com/eclipse-ee4j/mojarra/pull/4867.

**Comment by raupachz** [comment]
Hi @christophs78 ,

```
<dependency>
      <groupId>org.glassfish</groupId>
      <artifactId>jakarta.faces</artifactId>
      <version>3.0.0</version>
    </dependency>
```

**Comment by christophs78** [comment]
Yep. Please look at README.md of 3.0-branch. (old namespace)
https://github.com/eclipse-ee4j/mojarra/blob/3.0/README.md
I´ll do a PR to add a note regarding this to README.md of master. (It confused me a few day ago too.)

**Comment by raupachz** [comment]
Thanks a lot! Yes, maybe change the README. I did not check the closed issues and therefore missed it.
